src/vm/disassembler.cpp

```diff
@@ -78,16 +77,49 @@ void Disassembler::StaticInitialize()
 #if USE_COREDISTOOLS_DISASSEMBLER
     _ASSERTE(!IsAvailable());
 
-    // TODO: The 'coredistools' library will eventually be part of a NuGet package, need to be able to load
-    // that using appropriate search paths
-    LPCWSTR libraryName = MAKEDLLNAME(W("coredistools"));
-    HMODULE libraryHandle = CLRLoadLibrary(libraryName);
+	LPCWSTR libraryName = MAKEDLLNAME(W("coredistools"));
+  	// Try to load library
+	HMODULE libraryHandle = CLRLoadLibrary(libraryName);
+    
+#if !defined(_WIN32) 
```

Comment:
Thanks. I expected someone to tell me to use a better macro there :)

Yeah, as you said, the first CLRLoadLibrary will succeed on Windows if user places the library in the current executable directory. If user makes a mistake, the new code does what CLRLoadLibrary already tried which is just a little redundant. Even though we remove the distinction there, I think that we still need to distinguish between slash and backslash. I am wondering if we already have some utility function to handle it.